### ISSUE TITLE
Loss: nan WARNING:root:NaN or Inf found in input tensor. WARNING:root:NaN or Inf found in input tensor.

### ISSUE THREAD

**Issue by TatShri** [AshwinRJ/Federated-Learning-PyTorch#29]

**Comment by rui-ye** [comment]
Same issue. Anyone knows why?

**Comment by wyxzou** [comment]
I believe this issue is because NLLLoss is used but log_softmax is not used in some of the models. i.e. add F.log_softmax(out, dim=1) to CNNFashion_Mnist.

**Comment by Asomey** [comment]
> 

You're right. Thank you bro.
